toolchains/xbox/gxdk_xs_toolchain.cmake

```diff
@@ -21,8 +21,11 @@ set(CMAKE_TRY_COMPILE_PLATFORM_VARIABLES GDK_VERSION)
 set(CMAKE_SYSTEM_NAME WINDOWS)
 set(CMAKE_SYSTEM_VERSION ${GDK_VERSION})
 
-set(CMAKE_GENERATOR_PLATFORM "Gaming.Xbox.Scarlett.x64" CACHE STRING "" FORCE)
-set(CMAKE_VS_PLATFORM_NAME "Gaming.Xbox.Scarlett.x64" CACHE STRING "" FORCE)
+# set(CMAKE_GENERATOR_PLATFORM "Gaming.Xbox.Scarlett.x64" CACHE STRING "" FORCE)
```

Comment:
@vaind @jpnurmi 

I hacked this to test.
What would you recommend I do in order to have this work for both Xbox One and Xbox Series X|S?
I imagine we don't want two toolchain files, right? 

Can we have it toggle based on "architecture"? (`Gaming.Xbox.XboxOne.x64` vs `Gaming.Xbox.Scarlett.x64`)?

Once this is ready for review I'll ask Mischan for his input of course but trying to spare him until this is in better shape

Reply:
Passing in a variable from the command-line works too, but specialized toolchain files might be more convenient because you only need to select one toolchain file with help of auto-completion instead of passing an extra variable in addition to the toolchain file.

Reply:
These files rarely need to be touched. I propose taking a step back and following the original samples/structure by MS:

- https://github.com/microsoft/Xbox-GDK-Samples/blob/main/Samples/Tools/CMakeGDKExample/gxdk_toolchain.cmake
- https://github.com/microsoft/Xbox-GDK-Samples/blob/main/Samples/Tools/CMakeGDKExample/gxdk_xs_toolchain.cmake

Reply:
Thanks J-P!!